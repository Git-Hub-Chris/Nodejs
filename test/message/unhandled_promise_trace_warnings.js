'use strict';
const common = require('../common');
if (!process.execArgv.includes('--trace-warnings'))
  common.requireFlags(['--trace-warnings']);
common.disableCrashOnUnhandledRejection();
const p = Promise.reject(new Error('This was rejected'));
setImmediate(() => p.catch(() => {}));
