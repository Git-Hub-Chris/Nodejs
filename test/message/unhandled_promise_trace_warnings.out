(node:*) Error: This was rejected
    at * (*test*message*unhandled_promise_trace_warnings.js:*)
    at *
    at *
    at *
    at *
    at *
    at *
    at *
    at *
(node:*) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 1)
    at *
    at *
    at Promise.then *
    at Promise.catch *
    at Immediate.setImmediate [as _onImmediate] (*test*message*unhandled_promise_trace_warnings.js:*)
    at *
    at *
    at *
